src/Assets/Adic/Scripts/Framework/Binding/BindingFactory.cs

```diff
@@ -115,28 +115,52 @@ public class BindingFactory : IBindingFactory {
 		/// <param name="namespaceName">Namespace name.</param>
 		/// <returns>The binding condition object related to this binding.</returns>
         public IBindingConditionFactory ToNamespace(string namespaceName) {
-            return ToNamespace(namespaceName, BindingInstance.Transient);
+            return this.ToNamespace(namespaceName, BindingInstance.Transient, false);
 		}
-		
+
+		/// <summary>
+		/// Binds the key type to all assignable types in a given <paramref name="namespaceName"/> 
+		/// as transient bindings.
+		/// </summary>
+		/// <param name="namespaceName">Namespace name.</param>
+		/// <param name="includeChildren">Indicates whether child namespaces should be included.</param>
+		/// <returns>The binding condition object related to this binding.</returns>
+		public IBindingConditionFactory ToNamespace(string namespaceName, bool includeChildren)
+		{
```

Comment:
Adic's source code uses curly braces on the same line as declarations. I'll format it when merged.